src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -308,10 +308,12 @@ public int compare(FeedbackResponseAttributes o1, FeedbackResponseAttributes o2)
                 return order;
             }
 
-            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty() ? getNameForEmail(o1.recipientEmail)
-                                                                                    : getTeamNameForEmail(o1.recipientEmail);
-            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty() ? getNameForEmail(o2.recipientEmail)
-                                                                                    : getTeamNameForEmail(o2.recipientEmail);
+            String receiverTeam1 = getTeamNameForEmail(o1.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o1.recipientEmail)
+                                 : getTeamNameForEmail(o1.recipientEmail);
+            String receiverTeam2 = getTeamNameForEmail(o2.recipientEmail).isEmpty()
+                                 ? getNameForEmail(o2.recipientEmail)
+                                 : getTeamNameForEmail(o2.recipientEmail);
```

Comment:
I must admit, I've been feeling since long time ago that aligning `?` (and if I may add, `+` in many parts of the code) with the `=` is not right.